docker/maistra-proxy-builder_2.1.Dockerfile

```diff
@@ -9,7 +12,7 @@ RUN dnf -y upgrade --refresh && \
     dnf -y config-manager --set-enabled powertools && \
     dnf -y install git make libtool patch libatomic which \
                    autoconf automake libtool cmake python2 python3 \
-                   gcc gcc-c++ ninja-build golang annobin libstdc++-static \
+                   gcc-toolset-9 ninja-build golang \
```

Comment:
we also need `gcc-toolset-9-libatomic-devel`, build fails without it.

Reply:
done